pkg/apis/playlist/v0alpha1/mapper.go

```diff
@@ -0,0 +1,23 @@
+package v0alpha1
+
+import (
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+type NamespaceMapper = func(orgId int64) string
```

Comment:
Curious about your reasoning here - Why is `NamespaceMapper` a discrete type in playlists instead of a single, reusable method?  I think a single function (`namespaceFromCfg(cfg *setting.Cfg) string`) is all that's needed here, at which point we could move this function to the `setting` package, unless there are future plans for this.

Reply:
i'm torn on this one so not sure...  passing around setting.Cfg is pretty annoying 😢 

in practice this could be a global function, where the stackId is set globally, but that breaks the global variable rule

Reply:
I vote we leave it internal for now, and find a better pattern when we have n+1 :)